airbyte-integrations/connectors/source-google-search-console/source_google_search_console/manifest.yaml

```diff
@@ -675,6 +675,9 @@ definitions:
         error_handler:
           type: DefaultErrorHandler
           response_filters:
+            - type: HttpResponseFilter
```

Comment:
i'll note that this is only under the `search_analytics` and `search_by_keyword` streams (not `sites` or `sitemaps`, but those streams don't experience nearly the same level of traffic as the analytics streams